lib/chefspec/policyfile.rb

```diff
@@ -27,9 +27,12 @@ def setup!
         policyfile_path = File.join(Dir.pwd, "Policyfile.rb")
       end
 
+      Chef::WorkstationConfigLoader.new(nil).load
```

Comment:
I suppose this could be configurable through some rspec configuration, but I just added the bare minimum that gets it working by hunting and finding the default locations.